### DISCUSSION TITLE
Expo Snack Demo does not work.

### DISCUSSION THREAD

**Discussion by denis-j** [wix/react-native-ui-lib#2721] [Q&A]
Hello,

I just wanted to check out the expo snack demo, but it just doesn't work. On the website, you should get the same error as I do.

**Comment by AndreyEscobar** [comment]
I tried the expo snack demo and not work. 
So i tried to setup in a project and not work too. (https://wix.github.io/react-native-ui-lib/docs/getting-started/setup)
So i see a qrcode to download the expo demo (https://github.com/wix/react-native-ui-lib). The qrcode is invalid (There was a problem running the requested app.
